pallets/vc-management/src/lib.rs

```diff
@@ -368,5 +376,42 @@ pub mod pallet {
 			Self::deposit_event(Event::SchemaRevoked { account: sender, shard, index });
 			Ok(().into())
 		}
+
+		/// ---------------------------------------------------
+		/// The following extrinsics are supposed to be called by Sudo/Concuil only
+		/// This is the temporary function for manual operation of VCRegistry storage
+		/// ---------------------------------------------------
+		#[pallet::call_index(10)]
+		#[pallet::weight(195_000_000)]
+		pub fn add_vc(
```

Comment:
Maybe you missed this comment: https://github.com/litentry/litentry-parachain/issues/1544#issuecomment-1495589922

I think `add_vcregistry_item` is a better name, as we are not adding VCs here (we don't have VC content), but a VCRegistry item.